### PR TITLE
Cp UI refractor

### LINKED ISSUE #9: UI REFACTOR: Home Page
### Have you updated the title?

- [X] I have updated the issue's title.

### Discussion

Currently, the home page for the game looks like this:
Screenshot 2024-05-15 at 12 06 22 AM

While this is a nice layout, the UI doesn't necessarily resemble a game and feels very official. 

You have creative freedom here to retain the information about a commons and the join/visit functionality while improving the UI/UX of this page. 

Some ideas include:
- Implementing draggable cards for commons (being able to drag a commons you want to join over from the available commons side). For this see: https://www.geeksforgeeks.org/implement-drag-and-drop-using-react-component/
- Associating different colors for each commons
- Abstracting the id # column (not necessary information for users, UNLESS two commons have the same name. Could implement a check for this as well).

### Acceptance Criteria

- [x] Home page of application has an improved UI while still having a seamless user experience
- [ ] Information is displayed clearly and distinctly. 
- [x] Page is not confusing to navigate

### Implementation Todos

Frontend:
- [ ] Home page is modified according to UI changes

Testing:
- [ ] Front end tests pass and there is adequate coverage

### PR DESCRIPTION
**Pull request by charlene-patenaude**
## Overview

In this PR, Emily and Charlene added new fonts to the HomePage and Navbar to make it fit the theme of the website. We also adjusted the color on the animated-text to be a smooth transition instead of a harsh one. 

## Screenshots (Optional)
image

## Feedback Request (Optional)
Please give suggestions on the following:
- We are unsure about keeping the text animation seen in picture above - Currently it is an alternating rainbow/tomato (Emily) and we are unsure if it meshes well with the rest of the page/font 
- We are also considering removing the white boxes around the text but would like feedback

## Future Possibilities (Optional)
Some possible points could be:
- We would like to edit other UI elements on the project like the leaderboard and the stats but are pending approval from staff.

## Validation (Optional)
1. git fetch 
2. git checkout cp-UIRefractor
3. npm run storybook
4. check HomePage 

Dokku deploment : https://proj-happycows-et-dev.dokku-08.cs.ucsb.edu/

## Tests
NO Additional tests required
- [X] Frontend Unit tests (`npm test`) pass
- [X] Frontend Test coverage (`npm run coverage`) 100%
- [X] Frontend Mutation tests (`npx stryker run`) 100% 
- [X] Frontend Linting (`npx eslint --fix src`) 

## Linked Issues
Completes part of #9 (not fully finished)

### CODE REVIEW

**Review by Jumshim — APPROVED** [review]
~~Wow! Looks super good :)~~

~~LGTM!~~

File: frontend/src/main/pages/HomePage.css [hunk 1]
```diff
@@ -16,9 +21,14 @@
   text-fill-color: transparent;
   -webkit-background-clip: text;
   -webkit-text-fill-color: transparent;
-  animation: textclip 2s linear infinite;
+  animation: textclip 8s linear infinite;
   display: inline-block;
   font-size: 50px;
+  font-family: 'Rye', cursive;
+}
+.reg-text
```
Comment (Jumshim) [inline frontend/src/main/pages/HomePage.css]: ⛏️ I tried Ctrl+F'ing this section, and it doesn't seem like this CSS portion is used. Maybe take it out if it doesn't affect anything?

**Review by Jumshim — CHANGES_REQUESTED** [review]

**Comment by Jumshim** [comment]
LGTM!

**Review by Jumshim — APPROVED** [review]

**Review by AnikaArora — CHANGES_REQUESTED** [review]
Please link your dokku deployment so we can test it! Everything else looks good so far.

**Review by AnikaArora — DISMISSED** [review]
So it seems like you switched the boxes for join commons and visit commons. I think we should keep visit commons on the left and join commons on the right, unless you have a specific reason why you made that switch.

**Comment by charlene-patenaude** [comment]
> So it seems like you switched the boxes for join commons and visit commons. I think we should keep visit commons on the left and join commons on the right, unless you have a specific reason why you made that switch.

We did that because it makes more sense to move right to left like when we read. We felt as though it had a better flow than before.

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**